### PR TITLE
(progress) Proxy setup

### PR DESCRIPTION
**Pull request by CyanBlueCode**
proxy connected, removed views files, and general code clean up

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,11 +6,24 @@
     "start": "node ./bin/www"
   },
   "dependencies": {
+    "babel-eslint": "^10.0.3",
     "cookie-parser": "~1.4.4",
     "debug": "~2.6.9",
+    "eslint": "^6.7.0",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
```
Comment (shams-ali) [inline package.json]: theres no jsx in the server

File: package.json [hunk 2]
```diff
@@ -6,11 +6,24 @@
...
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.14.3",
```
Comment (shams-ali) [inline package.json]: theres no react

File: .gitignore [hunk 3]
```diff
@@ -22,3 +22,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+gkeys.json
```
Comment (shams-ali) [inline .gitignore]: you should be using .env

Comment (shams-ali) [reply .gitignore]: fix this in next PR

File: routes/index.js [hunk 4]
```diff
@@ -1,9 +1,51 @@
...
-var router = express.Router();
+const express = require('express');
 
-/* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+const router = express.Router();
+
+const { google } = require('googleapis');
+const keys = require('../gkeys.json');
```
Comment (shams-ali) [inline routes/index.js]: you should be using .env

File: routes/index.js [hunk 5]
```diff
@@ -1,9 +1,51 @@
...
+  );
+
+  async function gsGet(clientAuth) {
+    const gsApi = google.sheets({
+      version: 'v4',
+      auth: clientAuth
+    });
+
+    const options = {
+      spreadsheetId: '1JWYHiOightRfEg3qFrDniItyYII5fgXdi99m11mgSH0',
```
Comment (shams-ali) [inline routes/index.js]: this should be in .env

File: routes/index.js [hunk 6]
```diff
@@ -1,9 +1,51 @@
...
+
+    const sheetsData = {
+      projects,
+      events
+    };
+
+    return sheetsData;
+  }
+
+  client.authorize((err, tokens) => {
```
Comment (shams-ali) [inline routes/index.js]: should use promise or better yet...go learn "async await"

async await is the proper way to do async calls....please fix this in next PR

File: .gitignore (hunk 3 above)
Comment (CyanBlueCode) [reply .gitignore]: will fix in next PR

File: routes/index.js (hunk 4 above)
Comment (CyanBlueCode) [reply routes/index.js]: will fix in next PR

File: routes/index.js (hunk 5 above)
Comment (CyanBlueCode) [reply routes/index.js]: will fix in next PR

File: routes/index.js (hunk 6 above)
Comment (CyanBlueCode) [reply routes/index.js]: will fix in next PR

**Pull request merged**